Buildings/Resources/ReferenceResults/Dymola/Buildings_Fluid_HeatExchangers_Validation_WetCoilEffectivenessNTU.txt

```diff
@@ -0,0 +1,30 @@
+last-generated=2021-02-22
+statistics-initialization=
+{
+  "nonlinear": "0, 1, 1",
+  "numerical Jacobians": "0"
+}
+statistics-simulation=
+{
+  "linear": " ",
+  "nonlinear": "1, 1, 120, 1, 1, 1, 1",
```

Comment:
120x120 nonlinear system? Something went wrong here.

Reply:
Michael and Antoine, 

Would one of you let me know how to interpret those series of numbers? I have tried but didn't have time to dig in. Any documents that tell me how to read thoes numbers?

Reply:
This is due to the number of elements of the discretized coil model set to 30 as opposed to the default 4 (which reduces the size of the NL system to 16x16). 
This choice aims at reducing the modeling bias due to a low number of elements, see this plot:

![WetCoilEffectivenessNTU](https://user-images.githubusercontent.com/10965994/110546538-74c60c80-80e3-11eb-8180-a5f9f90f4ded.png)

I was tentative to adopt an intermediary value—say 10 or 20—but this is beneficial to the validation of our model. 

@kim1077 To know which variables are part of the NL system of equations with Dymola, you can select the option `Generate listing of translated Modelica code` in the `Translation` tab of the simulation set up widget, and look for `// Nonlinear system of equations` in the generated `dsmodel.mof` file.

Reply:
Antoine will change the static discretized model to a transient one and re-simulate.